src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/Android/CameraFragment.android.cs

```diff
@@ -538,32 +547,38 @@ async Task PrepareSession()
 				var surfaces = new List<Surface>();
 
 				// preview texture
-				if (texture.IsAvailable && previewSize != null)
+				if (previewSize != null && texture?.IsAvailable is true)
 				{
-					var texture = this.texture.SurfaceTexture;
+					var texture = this.texture.SurfaceTexture ?? throw new NullReferenceException();
 					texture.SetDefaultBufferSize(previewSize.Width, previewSize.Height);
 					var previewSurface = new Surface(texture);
 					surfaces.Add(previewSurface);
 					sessionBuilder.AddTarget(previewSurface);
 
 					// video mode
-					if (cameraTemplate == CameraTemplate.Record)
+					if (cameraTemplate is CameraTemplate.Record)
```

Comment:
Yup - they're the same. [`is` used to be more performant than `==`](https://stackoverflow.com/a/40676671/5953643), but Roslyn has has since been updated to make the behavior of the two operators the same when there is no overloaded equality operator.

I always prefer `is` over `==`, because `==` is an operator that can be overridden (e.g. `public static bool operator ==`). 

It's rare for anyone to override `==`, but someone could override `==` which could lead to bugs in the future.

Reply:
But if `==` is overridden, it's done for some reason. And one who did it, expects that that overridden logic will be used to compare two objects.

Anyway, we cannot override `==` for enums.